### DISCUSSION TITLE
"网址参数清理"可以清理时间?t=*的参数吗

### DISCUSSION THREAD

**Discussion by AquaMT** [the1812/Bilibili-Evolved#4766] [功能建议 / Ideas]
如题，首页的视频，鼠标移上去五秒以上，打开的链接就会有t=*的时间参数，此时想从头看就只能再退回开头。
另外，从历史记录页面点开看了一半的视频，即使没有t=*的参数也会自动定位到已经看过的进度，而从自定义顶栏-历史中打开的却有p=*和t=*的参数，这后面的参数可以也把它去掉吗

**Comment by Geeyun-JY3** [comment]
显然不能啊，清了还怎么空降到指定时间点，这个参数又不是只用于你提到的这些场景。
鼠标悬停首页视频五秒以上打开的链接带时间参数这个机制要说也是破站搞事，通过清理时间参数来解决这个机制带来的不便是捡了芝麻，丢了西瓜。
